storage/tick.go

```diff
@@ -85,9 +93,19 @@ func newTickManager(database database, opts Options) databaseTickManager {
 		c:        context.NewCancellable(),
 		tokenCh:  tokenCh,
 	}
+
+	runtimeOptsMgr := opts.RuntimeOptionsManager()
+	runtimeOptsMgr.RegisterListener(mgr)
```

Comment:
more a curiosity: is there a way to inspect the current runtime value in-use by the process (either log/metric)?

Reply:
Not currently no, it would be pretty trivial however to implement such a method as an RPC method (with an autogenerated HTTP/JSON endpoint).